Makefile

```diff
@@ -1,47 +1,33 @@
 # Unison file synchronizer: Makefile
 # See LICENSE for terms.
 
-.PHONY: all src
-
-default: text
-
-text:
-	$(MAKE) -C src UISTYLE=text
-
-test:
-	./src/unison -ui text -selftest
+.PHONY: all src docs manpage test depend clean install
 
 all: src manpage
 
 src:
 	$(MAKE) -C src
 
--include src/Makefile.ProjectInfo
-
-docs:
-	$(MAKE) -C src UISTYLE=text
+docs: src
 	$(MAKE) -C doc
 	$(MAKE) -C man
 
 manpage:
 	$(MAKE) -C man
 
-depend::
+test:
+	./src/unison -ui text -selftest
+
+depend:
 	$(MAKE) -C src depend
```

Comment:
I tentatively propose to remove `depend` completely from toplevel Makefile.

Reply:
As I see it, the fact that there is a makefile in src is a mere implementation detail, and the normal targets should work at top level.  There's also no good reason why docs couldn't end up with a depend; they are complicated enough.

Reply:
I agree that the end goal is not to have a makefile in src.

Reply:
I didn't say that was a goal.  Just that someone should be able to invoke make at top level to do the normal things and that should work.  But, taking depend out of top-level is out of scope for this PR, so we can worry about that later.